pkg/daemon/ceph/config/keyring.go

```diff
@@ -54,31 +51,38 @@ func WriteKeyring(keyringPath, authKey string, generateContents func(string) str
 	return writeKeyring(contents, keyringPath)
 }
 
-// CreateKeyring creates a keyring for access to the cluster with the desired set of privileges
-// and writes it to disk at the keyring path
-func CreateKeyring(context *clusterd.Context, clusterName, username, keyringPath string, access []string, generateContents func(string) string) error {
-	_, err := os.Stat(keyringPath)
-	if err == nil {
-		// no error, the file exists, bail out with no error
-		logger.Debugf("keyring already exists at %s", keyringPath)
-		return nil
-	} else if !os.IsNotExist(err) {
-		// some other error besides "does not exist", bail out with error
-		return fmt.Errorf("failed to stat %s: %+v", keyringPath, err)
-	}
-
-	// get-or-create-key for the user account
-	key, err := client.AuthGetOrCreateKey(context, clusterName, username, access)
-	if err != nil {
-		return fmt.Errorf("failed to get or create auth key for %s. %+v", username, err)
-	}
-
-	return WriteKeyring(keyringPath, key, generateContents)
-}
+// Only use is via device_test.go in osd
```

Comment:
This comment should be removed later. I'm pretty sure this isn't needed, but it's here in case anyone notices that it's actually necessary for some reason.

Reply:
i may actually need it for the ceph-volume work to call `createOSDBootstrapKeyring()`